.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "editor.trimAutoWhitespace": false,
+    "diffEditor.ignoreTrimWhitespace": false,
+    "editor.find.addExtraSpaceOnTop": false,
```

Comment:
Please remove this file, is unnecessary

Reply:
the vs code ide configuration file has been deleted.